Quick interim note on Secrets ignores (#1563)
semgrep/semgrep-docs: docs/semgrep-secrets/getting-started.md

* Wait for your scheduled Semgrep full scan
* Open a pull request or merge request and wait for Semgrep to scan the branch automatically

## Configure files to ignore

> Semgrep Secrets will scan files even if they are specified in a local `.semgrepignore`, since secrets can often be found in files that are not relevant for code scanning. To specify files to ignore in Semgrep Secrets as well:

Reviewer: ```suggestion
Semgrep Secrets scans all files, even those specified in a local `.semgrepignore` file, since secrets can often be found in files that are not relevant for code scanning. To specify files that Semgrep Secret should ignore:
```

Author: Semgrep Secrets :)

I'll commit something like both of these changes but I want to make the additional fixes as well

1. Sign in to Semgrep AppSec Platform.

> 2. Go to **Projects** and select the gear icon <i class="fa-solid fa-gear"></i> to access the settings for the related project.

Reviewer: ```suggestion
2. Go to **Projects** and find your project. Select the gear icon <i class="fa-solid fa-gear"></i> to access the settings for the related project.
```

> 3. Add entries to the **Path ignores** box to ignore them for Semgrep Secrets as well as other Semgrep products.

Reviewer: ```suggestion
3. Add the file paths to the **Path ignores** box. Semgrep ignores all files listed.
```

Author: Hmm I think we should probably say "all paths listed"
